tracer/src/Datadog.Trace/RemoteConfigurationManagement/Transport/RemoteConfigurationApi.cs

```diff
@@ -89,6 +89,14 @@ public static RemoteConfigurationApi Create(IApiRequestFactory apiRequestFactory
                 return null;
             }
 
+            var contentType = apiResponse.GetHeader("content-type");
+            if (!string.Equals(contentType, MimeTypes.Json))
+            {
+                var content = await apiResponse.ReadAsStringAsync().ConfigureAwait(false);
+                Log.Warning("Failed to receive remote configurations: expected " + MimeTypes.Json + " but received {ContentType}: {ResponseContent}", contentType, content);
```

Comment:
I wonder if we should cap the size. At this point we're displaying an unidentified payload, it could be arbitrarily large

Reply:
Yeah, cut to the first 128 bytes, should be enough to know what is happening and avoid spamming.
Could it contain special characters? does it need escaping?

Reply:
> Could it contain special characters? does it need escaping?
Yes, and no, that's handled by the JSON serializer 🙂

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/5652/commits/d5b1a4de37562d5f8c58f00fee3998922ec5a835